### PR TITLE
New bootstrap fixes

### PR DESCRIPTION
**Pull request by K900**
This gets things building, at least. We should probably figure out how to make these conditional on nixpkgs version though...

### CODE REVIEW

File: overrides/default.nix [hunk 1]
```diff
@@ -2760,14 +2760,6 @@ lib.composeManyExtensions [
         }
       );
 
-
-      # Stop infinite recursion by using bootstrapped pkg from nixpkgs
```
Comment (FRidh) [inline overrides/default.nix:2764]: those using poerty2nix with python2 projects may want to keep this

**Comment by tjni** [comment]
I would be happy to help fix what's broken too. I should have checked this before merging, but I forgot. With that said, I might need some hand holding since I don't use poetry2nix.

File: flake.lock [hunk 2]
```diff
@@ -40,16 +40,16 @@
...
     "nixpkgs": {
       "locked": {
-        "lastModified": 1689192006,
-        "narHash": "sha256-QM0f0d8oPphOTYJebsHioR9+FzJcy1QNIzREyubB91U=",
+        "lastModified": 1693156971,
+        "narHash": "sha256-BE6BIXmVQyrOhkX4s/nP+/Xeipb1rRS1qLK/kpe7Pqk=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "2de8efefb6ce7f5e4e75bdf57376a96555986841",
+        "rev": "f900e733c1602b4c46064a9ef0514bb2345a8949",
```
Comment (lucc) [inline flake.lock:47]: I would say this makes it "conditional on nixpkg version". Otherwise there is `lib.version` which is a string :/

**Comment by takeda** [comment]
What is the status of the fix? Do I need to include this patch if I want to use latest version of nixpkgs and poetry2nix?

**Comment by delroth** [comment]
> What is the status of the fix? Do I need to include this patch if I want to use latest version of nixpkgs and poetry2nix?

You'll want to follow #1291 which has more info and status updates.

**Comment by K900** [comment]
I really don't have the time or the spoons to invest into this anymore. Anyone interested in picking this up, please do.

**Comment by K900** [comment]
I'll just close the PR because I really don't intend to work on this any time soon. Anyone willing to pick up the work is more than welcome to.

**Pull request closed without merging**